crt_portal/cts_forms/tests/test_forms.py

```diff
@@ -750,6 +750,16 @@ def test_post_with_all(self):
             self.assertEquals(last_activity.description, 'a comment')
             self.assertEquals(last_activity.actor, self.user)
 
+    def test_post_with_email_count_sort(self):
+        ids = [report.id for report in self.reports]
+        self.post(ids, all_ids=True, confirm=True, return_url_args='sort=-email_count', comment='email count sort', status='closed')
```

Comment:
It doesn't look like this fails as expected without the associated changes to `forms.py`.

Using `self.post` here passes `'next': '?per_page=15'` in the request resulting in a queryset that's not ordered by `email_count` and doesn't cause the `Cannot resolve keyword...` exception.

Reply:
🤔 This test does fail for me as expected when the changes in `forms.py` (line 1489 specifically) are taken out:

![image](https://user-images.githubusercontent.com/1425377/107787651-01fc8800-6d15-11eb-9c31-bed886195de1.png)

Reply:
🤔 Ahhhh, double checking on my end

Reply:
My mistake @timoballard, I appreciate the quick follow-up and leading me to the other changes in `test_forms.py` that I missed when isolating and checking this test!